modules/tinymce/src/core/test/ts/browser/FormatterApplyTest.ts

```diff
@@ -2269,15 +2269,15 @@ describe('browser.tinymce.core.FormatterApplyTest', () => {
 
   it('Apply format to node outside fake table selection', () => {
     const editor = hook.editor();
-    editor.setContent('<p>test</p><table><tbody><tr><td data-mce-selected="1">cell 1</td><td>cell 2</td></tr><tr><td data-mce-selected="1">cell 3</td><td>cell 4</td></tr></tbody></table>');
-    LegacyUnit.setSelection(editor, 'td', 0, 'td', 0);
+    editor.setContent('<p>test</p><table><tbody><tr><td>cell 1</td><td>cell 2</td></tr><tr><td>cell 3</td><td>cell 4</td></tr></tbody></table>');
```

Comment:
Perhaps we should then implement a method to trigger the fake selection for tables instead of trying to fake it with the `data-mce-selected`-tags?

Reply:
There is a TableTestUtils maybe we can stick in some form of select cell feature there. The problem with cell selection is that it's supposed to be smart about mixed colspans and rowspans etc but maybe that can be improved later. I would want to avoid having to go though some table model to select things. Maybe it could take a start x, y and and end x, y but those coords are non cell/rowspan normalized so y would be the TR index and X would be the TD/TR cell index.

Reply:
Do you have any idea for how best to do it? I'm not too familiar with how the fake cell selection works.